db/migrate/20200514091507_make_page_layoutpage_null_false.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class MakePageLayoutpageNullFalse < ActiveRecord::Migration[5.2]
+  def change
+    change_column :alchemy_pages, :layoutpage, :boolean, null: false
```

Comment:
We probably want to [use `change_column_null` here](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_null) and provide a default of `false` as last parameter